docs-src/source/data_types.rst

```diff
@@ -1,6 +1,36 @@
 Data Types in Pyrlang
 =====================
 
+Things to be aware about
+------------------------
+
+As we started this we've seen that there are some things that doesn't map
+that good between python and erlang, and that the best solution for it could
+be different pending on the problem you're currently trying to solve.
+
+Erlang strings is more of syntactic sugar, but it is infact a list of
```

Comment:
Erlang strings are lists of `char()` where `char()` is Unicode symbol range from like 1 to 0x1fsomething000, 0..255 are ASCII strings which match Erlang strings too. Erlang program can send you a Unicode string which `term` will encode as a list of small and large integers and it is responsibility of Pyrlang user to know whether that is a unicode string or a list of integers.

Reply:
yea but when using generic calls there is a problem with the round trip, but maybe this is can be fixed in a way I don't know about.

In the documentation it's even stated that string() isn't a data type http://erlang.org/doc/reference_manual/data_types.html#string. Char is described under number http://erlang.org/doc/reference_manual/data_types.html#number.

So there exists some friction here I'm just saying we need to look at what works best most of the time.

so when sending something to erlang sending `[104,101,108,108,111]` and not getting `"hello"` or vice versa is an issue. 

We've starting to use it now and it is problematic to get code going with the interpretation that is right now. We can get around it with explicitness, but the idea with Pyrlang was to try and make the interaction as smooth as possible.
an